docs/source/python_api/mlflow.environment_variables.rst

```diff
@@ -0,0 +1,6 @@
+mlflow.environment_variables
+============================
+
+.. automodule:: mlflow.environment_variables
+    :members:
+    :undoc-members:
```

Comment:
@WeichenXu123 @harupy The docs currently make it a bit difficult to see the string name of the environment variable:

<img width="939" alt="Screen Shot 2022-07-05 at 11 19 46 PM" src="https://user-images.githubusercontent.com/39497902/177482380-c0907448-1019-4ed3-b903-c259eb26ea07.png">


The user can probably infer that it's the same as the variable in the `mlflow.environment_variables` module, but it would be nice if we could render it as:

```
mlflow.environment_variables.MLFLOW_HTTP_REQUEST_BACKOFF_FACTOR = "MLFLOW_HTTP_REQUEST_BACKOFF_FACTOR"
...
```

etc. Is this possible by adding a `__repr__` method to the environment variable class or by making some tweaks to the sphinx config for this page?

Reply:
@dbczumar 

```diff
diff --git a/mlflow/utils/env.py b/mlflow/utils/env.py
index 0e3de2ade..08ff5810d 100644
--- a/mlflow/utils/env.py
+++ b/mlflow/utils/env.py
@@ -29,3 +29,6 @@ class EnvironmentVariable:
 
     def __str__(self):
         return f"Environment variable: name={self.name}, type={self.type}, default={self.default}"
+
+    def __repr__(self):
+        return self.__str__()
```

produces:

<img width="1070" alt="Screen Shot 2022-07-06 at 16 12 41" src="https://user-images.githubusercontent.com/17039389/177491467-48cfb03c-7612-4e7c-bd47-300a2241b25d.png">

Reply:
Relevant code in sphinx: https://github.com/sphinx-doc/sphinx/blob/12e86ff0e102efb3e33f2cab31cce5539450b735/sphinx/util/inspect.py#L425-L432